ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +405,186 @@ def test_trustdomain_disable_disables_subdomain(self):
                 assert 'no such user' in res.stderr_text
         # verify the user can be retrieved after re-enabling trustdomain
         self.master.run_command(['id', user])
+
+
+class TestNestedMembers(IntegrationTest):
+    num_clients = 1
+    username = "testuser001"
+    userpasswd = 'Secret123'
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+        tasks.install_client(cls.master, cls.clients[0])
+
+    @pytest.fixture
+    def nested_group_cleanup(self):
+        """Clean up groups and user created"""
```

Comment:
The fixture can be written like this:
```
@pytest.fixture
def nested_group_cleanup(self):
    # Do setup tasks, user creation, group creation etc...
    yield
    # Do cleanup tasks
```
so that the test gets executed after the setup tasks have been called and before the clean up tasks.

Reply:
modified as per suggestion